content/rs/administering/cluster-operations/updating-certificates.md

```diff
@@ -5,51 +5,83 @@ weight: $weight
+The certificates that you can replace are:
 
-- Use the REST API to replace the certificate:
+- API - The certificate used by the API
+- CM - The certificate used by the management user interface or the cluster manager and the Sentinel discovery service
+- Proxy - The certificate used to establish communication between a client and the databases
+- Syncer - The certificate used to encrypt replication traffic between clusters
+- Metrics exporter - The certificate used by Prometheus
 
-    ```src
+### Using the CLI
 
-    curl -k -X PUT -u "<username>:<password>" -H "Content-Type: application/json" -d '{ "name": "<cert_name>", "key": "<key>", "certificate": "<cert>" }' https://<cluster_address>:9443/v1/cluster/update_cert
+To replace certificates using the rladmin CLI, run:
 
-    ```
+```src
+ rladmin cluster certificate set <cert-name> certificate_file <cert-file-name>.pem key_file <key-file-name>.pem
+```
+
+Where:
+
+- cert-type - The type of certificate you want to replace:
+    - For management UI: `cm`
+    - For REST API: `api`
+    - For database endpoint: `proxy`
+    - For syncer: `syncer`
+    - For metrics exporter: `metrics_exporter`
+- cert-file-name - The name of your certificate file
+- key-file-name - The name of your non-password protected key file
+
+For example, to replace the cm certificate with the private key "key.pem" and the certificate file "cluster.pem":
+
+```src
+rladmin cluster certificate set cm certificate_file cluster.pem key_file key.pem
+```
+
+### Using the REST API
 
-    Where:
+To replace a certificate using the REST API, run:
+
+```src
+curl -k -X PUT -u "<username>:<password>" -H "Content-Type: application/json" -d '{ "name": "<cert_type>", "key": "<key>", "certificate": "<cert>" }' https://<cluster_address>:9443/v1/cluster/update_cert
+```
 
-    - cert_name - The name of the certificate to replace:
-        - For management UI: `cm`
-        - For REST API: `api`
-        - For database endpoint: `proxy`
-        - For syncer: `syncer`
-    - key - The contents of the *_key.pem file
+Where:
 
-    {{% tip %}}The key file contains `\n` end of line characters (EOL) that you cannot paste into the API call. You can use `sed -z 's/\n/\\\n/g'` to escape the EOL characters.{{% /tip %}}
+- cert_type - The name of the certificate to replace:
+    - For management UI: `cm`
+    - For REST API: `api`
+    - For database endpoint: `proxy`
+    - For syncer: `syncer`
+    - For metrics exporter: `metrics_exporter`
+- key - The contents of the *_key.pem file
 
-    - cert - The contents of the *_cert.pem file
+    {{% tip %}}
+    The key file contains `\n` end of line characters (EOL) that you cannot paste into the API call.
+    You can use `sed -z 's/\n/\\\n/g'` to escape the EOL characters.
+    {{% /tip %}}
 
-    The certificate is copied automatically to all nodes in the cluster.
+- certificate - The contents of the *_cert.pem file
```

Comment:
In the example it's "cert" and not "certificate"

Reply:
👍